repo/packages/M/msoms/2/marathon.json.mustache

```diff
@@ -0,0 +1,69 @@
+{
+  "id": "{{msoms.app_id}}",
+  "cpus": 1,
+  "mem": 256,
+  "disk": 0,
+  "instances": {{msoms.instances}},
+  "container": {
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/var/run/docker.sock",
+        "hostPath": "/var/run/docker.sock",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/var/lib/docker/containers",
+        "hostPath": "/var/lib/docker/containers",
+        "mode": "RW"
```

Comment:
This is the change with respect to the mount path. This is the only update to the file.